src/event.c

```diff
@@ -53,6 +54,15 @@ int init_event_system(struct daemon_conf *config)
 	return 0;
 }
 
+void flush_cache(struct daemon_conf *config)
+{
+	msg(LOG_DEBUG, "Flushing caches");
+	destroy_lru(subj_cache);
```

Comment:
i wouldn't think that its necessary to flush the subject cache. The subjects are processes. A database update shouldn't affect out notion of what we know about the process.